src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/BaseRecalibratorDataflow.java

```diff
@@ -250,7 +250,8 @@ private static CountingReadFilter readFilter( final SAMFileHeader header ) {
     private void checkSequenceDictionaries(final SAMSequenceDictionary refDictionary, SAMSequenceDictionary readsDictionary) {
         Utils.nonNull(refDictionary);
         Utils.nonNull(readsDictionary);
-        SequenceDictionaryUtils.validateDictionaries("reference", refDictionary, "reads", readsDictionary, true);
+        // temporary hack so JP can keep on working
```

Comment:
Could you make this comment say exactly what the hack is i.e "set SequenceDictionaryValidation to false" so that someone has a hope of understanding it in the future when we forget to ever change it back?